src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -367,4 +369,88 @@ public boolean contains(String key) {
 
         return mPrefs.contains(key);
     }
+    
+    public TokenCacheItem getFamilyRefreshTokenItemForUser(String userId) {
+        Iterator<TokenCacheItem> results = this.getAll();
```

Comment:
why you need getAll? Family RT is stored as a separate entry with key as User and Authority.